common/borders/borders-en.html

```diff
@@ -0,0 +1,77 @@
+---
+{
+	"title": "Borders",
+	"language": "en",
+	"altLangPage": "borders-fr.html",
+	"breadcrumbs":
+	[
+		{
+			"title": "GCWeb home",
+			"link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html"
+		}
+	],
+	"secondlevel": false,
+	"dateModified": "2023-06-20",
+	"share": "true"
+}
+---
+
+<h3>CSS Class</h3>
+<dl class="dl-horizontal">
+	<dt><code>.brdr-left</code></dt>
+	<dd>Set a border to the left of the element</dd>
+	<dt><code>.brdr-tp</code></dt>
+	<dd>Set a border on top of the element</dd>
+	<dt><code>.brdr-rght</code></dt>
+	<dd>Set a border to the right of the element</dd>
+	<dt><code>.brdr-bttm</code></dt>
+	<dd>Set a border below the element</dd>
+	<dt><code>.brdr-0</code></dt>
+	<dd>Remove borders</dd>
+	<dt><code>.brdr-rds</code></dt>
+	<dd>Remove border radius</dd>
+</dl>
+<div class="row">
+	<div class="col-md-6">
+		<div class="panel panel-default">
+			<div class="panel-body">
+				<h3 class="mrgn-tp-0">Appearance</h3>
```

Comment:
Considering the page start with an h1 (heading level 1) created by jekyl by using the "title" property in your front-matter. This heading would need to be changed for an heading level 2.

```suggestion
				<h2>Appearance</h2>
```

(No need to remove the top margin here. FYI - If it was used in panel, then the panel title must be moved into the panel heading section.)